changelog/645.doc.rst

```diff
@@ -0,0 +1 @@
+Fix the grammatical error in :class:`Guild` channel properties.
```

Comment:
shouldn't this be plural?
```suggestion
Fix the grammatical errors in :class:`Guild` channel properties.
```

Reply:
https://github.com/DisnakeDev/disnake/pull/645/commits/080cfbd0a2ab0dedcda6f1b1a69ddc17d90828e8